### ISSUE TITLE
input files

### ISSUE THREAD

**Issue by 0Mrbit1** [0Mrbit1/Minishell#5]
input files can be many , and all of them needs to be parsed  .

**Comment by anas-jmp** [comment]
what do you talking about  " input files can be many , and all of them needs to be parsed ."
input file are handled bor !!!

**Comment by anas-jmp** [comment]
yeah yeah i see now  , the solution is to change it to an array for multiple input files
